jetty-core/jetty-util/src/main/java/org/eclipse/jetty/util/resource/PathResource.java

```diff
@@ -142,26 +148,34 @@ public static boolean isSameName(Path pathA, Path pathB)
 
     PathResource(URI uri, boolean bypassAllowedSchemeCheck)
     {
-        // normalize to referenced location, Paths.get() doesn't like "/bar/../foo/text.txt" style references
-        // and will return a Path that will not be found with `Files.exists()` or `Files.isDirectory()`
-        this(Paths.get(uri.normalize()), uri, bypassAllowedSchemeCheck);
+        this(Paths.get(uri), uri, bypassAllowedSchemeCheck);
```

Comment:
Nice!!! no normalize in the constructor!!!

Reply:
Nor normalization in output of filenames, or normalization in the resolveAlias() either.